src/componentsNext/FilePicker/FilepathSegments.tsx

```diff
@@ -0,0 +1,45 @@
+import { Breadcrumb, BreadcrumbRoot } from "~/design/Breadcrumbs";
+
+import { FC } from "react";
+import { Home } from "lucide-react";
+
+export type FilePathSegmentsType = {
```

Comment:
As suggested in `src/componentsNext/FilePicker/FilepickerItem.tsx`: We might rename them to `FilePathSegmentsProps`